TF_Application/Assets/Examples/FillCursor/Scripts/FillCursor.cs

```diff
@@ -24,6 +24,14 @@ public class FillCursor : TouchlessCursor
 
     public Image fillRingImage;
 
+    // Variable: minRingThickness
+    // The minimum thickness the ring can be.
+    public float minRingThickness = 1.05f;
```

Comment:
Also much like the comment on the DotCursor, these two should maybe be overrides just to save any minor access issues

Reply:
> Should these maybe be public but readonly? Is there a situation where we want people to be able to edit this live?

For Tooling users, I think these values shouldn't be readonly. Some may want very thick or very thin cursor rings, the min/max defaults here are tailored towards TouchFree App

Reply:
Yeah but you can override them in an inheriting class, which lets you set your own values. The readonly thing more affects whether you can tweak it live. I guess because you can't loosen access modifiers when overriding though this might be better to leave open so we don't restrict customer cursors.